app/scripts/deploy_service.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Deploys a Hivemind contract to localhost:8546
+ *
+ * Usage: node deploy.js <model_config file path> <deposit_amount>
+ */
+
+const chalk = require('chalk');
+const oasis = require('@oasislabs/client');
+const fs = require('fs');
+
+var multisig_deploy = oasis.workspace.Multisig.deploy(
+  ['b8b3666d8fea887d97ab54f571b8e5020c5c8b58', 'ff8c7955506c8f6ae9df7efbc3a26cc9105e1797'], 2,
+  { header: {confidential: false} }
+);
+
+multisig_deploy.then(async (multisig) => {
+  var multisig_address = Buffer.from(multisig._inner.address).toString('hex');
+  fs.writeFileSync('multisig_address', '0x' + multisig_address);
```

Comment:
do you mean to write these into `.oasis`?

Reply:
No I meant to write them to those file paths.